src/Kafka/ManageKafkaPermissions/components/ViewAccountDetails.tsx

```diff
@@ -0,0 +1,46 @@
+import { Text, TextContent, TextVariants } from "@patternfly/react-core";
+import { Trans, useTranslation } from "react-i18next";
+import type { AclBinding } from "../types";
+import { ReviewPermissionsTable } from "./ReviewPermissionsTable";
+
+export type ViewAccountDetailsProps = {
+  accountId: string | undefined;
+  existingAcls: AclBinding[];
+  onRemoveAcl: (index: number) => void;
+};
+
+export const ViewAccountDetails: React.VFC<ViewAccountDetailsProps> = ({
+  accountId,
+  existingAcls,
+  onRemoveAcl,
+}) => {
+  const { t } = useTranslation(["manage-kafka-permissions"]);
+  const HelperText: React.FunctionComponent = () => {
+    if (accountId === "*") {
+      return t("all_accounts_help");
+    }
+    return (
+      <Trans ns={["manage-kafka-permissions"]} i18nKey="selected_account_help">
```

Comment:
we can handle the translation link this also   <Trans
          ns={"manage-kafka-permissions"}
          i18nKey={"selected_account_help"}
          components={{ bold: <strong /> }}
        />